pkg/server/server.go

```diff
@@ -2,22 +2,21 @@ package server
 
 import (
 	"fmt"
-	"html/template"
-	"io"
-	"net/http"
-	"net/url"
-	"os"
-	"path"
-
 	"github.com/coreos/dex/api"
 	"github.com/coreos/pkg/capnslog"
 	"github.com/coreos/pkg/health"
-
 	"github.com/openshift/console/pkg/auth"
 	helmhandlerspkg "github.com/openshift/console/pkg/helm/handlers"
 	"github.com/openshift/console/pkg/proxy"
 	"github.com/openshift/console/pkg/serverutils"
+	"github.com/openshift/console/pkg/terminal"
 	"github.com/openshift/console/pkg/version"
+	"html/template"
```

Comment:
imports moved, but originally were in the correct order.

Reply:
Sorry, if I'm not mistaken - it's done by `go fmt`. Will revert this change